### ISSUE TITLE
Not really an issue

### ISSUE THREAD

**Issue by leriaz** [xNNism/LG-G3_Nethunter#3]
Hey sorry for opening an issue but I didn't know where else to find you. 
I just moved from Fulmics to CM13 and I thought I'd give Nethunter a shot. 
Then I noticed this: "sadly current Branches of the CM13 Kernel doesnt allow to install Nethunter, im working on a CAF release rightnow."

Is this an old message so I just go ahead and flash nethunter, or is this a current issue?

**Comment by Fearup** [comment]
its not an old message wait for the new release if you can find an old CM13 nightly flash that then flash Nethunter im not sure what will happen if you flash Nethunter right now but i have used it before and it all worked only thing that didnt work was those USB attacks

**Comment by leriaz** [comment]
I see. Well then I'd rather wait and see how it goes. Thanks.

**Comment by xNNism** [comment]
if you flash Nethunter on the newest nightly you get a bootloop because of some Camera driver issue, idk why. Maybe i switch the main Rom to a CAF release this weekend. All newer Cyanogenmod kernels are failing with a Camera driver issue and i dont know why :(

**Comment by xNNism** [comment]
If there is need i can try to compile a kernel for Fulmics (or any other rom) with nethunter support, just leave a msg here..

**Comment by leriaz** [comment]
Nah, it's fine. I'd rather not go back to Fulmics. If you happen to get it stable on CM that's cool, otherwise still fine, no pressure. Thanks though.

**Comment by kauegimenes** [comment]
Any update on this Issue?

**Comment by Fearup** [comment]
Fulmics and stock are laggy af cm + nethunter was really awesome and i feel the new build will be even better if you ask me keep it for cm only no need to waste time on a port for a rom that laggs.

**Comment by xNNism** [comment]
Yeah allright, i totally agree with you.

 atm i test some kernel sources for CM13 and 1-2 are really encouraging. maybe i shoot one up to test later this evening...and plz always take a backup, of course i test every build on a bunch of phones but who knows....

**Comment by leriaz** [comment]
Just flashed the new release. Still updating so can't say much.
The kali boot animation is cropped but that's irrelevant.
Once it's done updating I'll test things and update the post here with an issues that may occur.
